kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/server/GwtComponentServiceImpl.java

```diff
@@ -59,6 +62,8 @@
     private static final String SERVICE_FACTORY_PID = "service.factoryPid";
     private static final String KURA_UI_SERVICE_HIDE = "kura.ui.service.hide";
 
+    private static final int SERVICE_WAIT_TIMEOUT = 60000;
```

Comment:
This here is usually the place for constants. Besides, you don't want to have to initialize the variable every time you call the method.